proj/src/store.js

```diff
@@ -0,0 +1,18 @@
+import { createStore, applyMiddleware, compose } from 'redux';
+import rootReducer from 'reducers/rootReducer'
+
+const customMiddleWare = store => next => action => {
```

Comment:
try to call the middleware according to the purpose it serves. 'customMiddleWare' doesn't imply on what's the middleware's job